src/parser/payment-module.ts

```
@@ -20,6 +20,7 @@ import Decimal from "decimal.js";
 import { BigNumber, ethers, utils } from "ethers";
 import type { Database } from "../adapters/supabase/types/database";
 import { PaymentConfiguration, paymentConfigurationType } from "../configuration/payment-configuration";
+import { DifferentialRewardTracker } from "./differential-rewards";
```

ESLint

[error] 'DifferentialRewardTracker' is defined but never used. Allowed unused vars must match /^_/u.

(`@typescript-eslint/no-unused-vars`)

---

[error] Remove this unused import of 'DifferentialRewardTracker'.

(sonarjs/unused-import)